### ISSUE TITLE
Where to provide 2FA to backup Bans and Auto Moderation Rules

### ISSUE THREAD

**Issue by SENPAY98K** [OutwalkStudios/discord-backup#33]
WARNING] 2FA Required for Auto Moderation Rules
[WARNING] 2FA Required for Bans

What should we do for it ?

**Comment by jleeson** [comment]
@SENPAY98K, if you simply want to remove the warning, you can use the new `ignore2FA` option, otherwise the bot owner needs to have 2FA enabled in order to back up the auto moderation rules and bans.

**Comment by SENPAY98K** [comment]
But its a bot! 
how is it possible to enable 2fa

**Comment by SENPAY98K** [comment]
I am the bot owner and i have 2fa enabled . I think there is something else missing here

**Comment by jleeson** [comment]
Sorry about that, I just checked myself and this is a bug in the implementation. The author of the PR forgot to check if the server had 2FA requirements enabled or not, given what you said there may be another issue as well. We are looking into it and will respond here when a fix is available.

**Comment by SENPAY98K** [comment]
Okay, thank you.

**Comment by jleeson** [comment]
@SENPAY98K v0.7.5 has been released and contains a fix for this feature. Hope that helps!

**Comment by SENPAY98K** [comment]
Hello, i have tried it and i did not get any warning, but also none bans are saved.

**Comment by jleeson** [comment]
Interesting, Im gonna take over fixing this moving forward. Can you provide the settings you are providing to the backup.create function?

**Comment by SENPAY98K** [comment]
![image](https://github.com/user-attachments/assets/5a21a2da-659b-4d91-b7c3-80083861fecd)

I will also check emojis since previously were giving error due to a property change

**Comment by jleeson** [comment]
Upon testing it looks like there is another bug in this package. Everything works when 2FA is disabled on the server and owner account but when using 2FA, it fails to back up the roles and auto mod rules still. I will continue investigating this and push a fix when possible.

**Comment by SENPAY98K** [comment]
Roles ? for me it did backup the roles

**Comment by jleeson** [comment]
Sorry, I meant bans. my bad.

**Comment by SENPAY98K** [comment]
Also emojis backup works fine

**Comment by jleeson** [comment]
@SENPAY98K I apologize for the inconvenience on this. I took over fixing this and have personally testing everything to ensure this latest fix solves it. There was a misnamed property name buried in a verbose conditional return causing the issue. If you update to v0.7.6, it should be solved. I have confirmed this with my own server testing with all combinations of 2FA being enabled/disabled.

If for whatever reason there is another issue, I can reopen this.
